cmd/virtual-kubelet/root/root.go

```diff
@@ -12,17 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package root provides methods to build and start the virtual-kubelet.
 package root
 
 import (
 	"context"
-	"fmt"
 	"os"
 	"path"
 	"time"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	module "github.com/virtual-kubelet/virtual-kubelet/node"
```

Comment:
The `module` name override could be probably removed.

Reply:
Yes, it could. I left it to minimise the number of changes and make it easier for the reviewer.

I can do the change just before merging to master.

Reply:
As you prefer, it seems to me there are just a few occurrences.